### ISSUE TITLE
Document target JSON Schema draft

### ISSUE THREAD

**Issue by rafamel** [json-schema-faker/json-schema-faker#484]
Hi guys. I was looking for the JSON Schema draft this library targeted and was unable to find it in the documentation. As far as I could gather from the issues, it targets Draft 04. It would be great if this could be added to the documentation or, otherwise, if it already is and I somehow missed it, make it more prominent.

Thanks for the awesome work!

**Comment by pateketrueke** [comment]
I don't get you, can you explain by example?

**Comment by rafamel** [comment]
Hi Alvaro, sure. I was just putting forwards it would be great to explicitly state in the README/docs that json-schema-faker targets JSON Schema draft 04.

**Comment by pateketrueke** [comment]
Oh yeah, sure, sorry for that.

Currently, v0.4.x targets only draft-04, but there are plans to support more drafts.

Our pending [RC17](https://github.com/json-schema-faker/json-schema-faker/pull/481) implements few stuff from recent drafts.

It is also a issue #289 for that but I'm pretty sure we don't have enough feature to fulfill it.

Please, if you think we should state this on both releases let me know.

Also, if you want to test recent features please fork and test by hand. 👍

**Comment by rafamel** [comment]
Hi Alvaro. I would imagine a common use case for some people would be to generate data against the OpenAPI spec. OpenAPI uses a subset of draft 00, which means libs like `openapi-schema-to-json-schema` must be used or manual modifications made for schemas to be used with this lib. I do think the targeted draft should be made explicit, but I don't want to intrude in how you manage the project -great work by the way!!

In terms of implementation, I would opt for something like what validators like `ajv` and `djv` do: [have a default target draft, but allow to specify the targeting of a different draft](https://github.com/epoberezkin/ajv#using-version-6) instead of implementing everything as part of the core, as there might be conflicts between specs, and in the case the whole spec for a supported draft is not implemented, but just a subset, making it explicit in the docs.

Hope it helps clarify what I was putting forwards!

**Comment by pateketrueke** [comment]
Yeah, atm is hard to achieve the multi-draft support while keeping this up-to-date. Today we're getting outdated and `json-schema-faker` is considered obsolete, see #289 -- right now I want to stabilize this and then add support for more drafts.

If using `openapi-schema-to-json-schema` is enough to starting to work with our current release/next-RC I don't think is an urgency to me to include that as built-in, I hope end users are able to transform their schemas into something compatible.

This can change over time, but right now we want to stay relevant and usable before anything more satisfying for most user-needs.

Partial support will be stated on the README yet.

I'll be working on this asap, thank you!

**Comment by pateketrueke** [comment]
Closing — v0.6.0 documents supported JSON Schema drafts. The library targets Draft 07 as baseline with 2019-09 / 2020-12 keywords supported where implemented.
